gui/gui/src/TGCommandPlugin.cxx

```diff
@@ -276,6 +276,14 @@ Bool_t TGCommandPlugin::HandleTimer(TTimer *t)
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Let user stop the internal timer when there is no need to check for remote.
+
+void TGCommandPlugin::StopTimer()
```

Comment:
What about `ToggleTimer()` to turn it on/off? (just wondering)

Reply:
Good idea, applied. :)